internal/facilities/retrieve.go

```diff
@@ -38,22 +39,22 @@ func (c *Client) Retrieve() *cobra.Command {
 
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cmd.SilenceUsage = true
-			facilities, _, err := c.Service.List(c.Servicer.ListOptions(nil, nil))
+			facilities, _, err := c.Service.FindFacilities(context.Background()).Execute()
```

Comment:
If you rename `facilities` to `facilitiesList` here, you can do `facilities := facilitiesList.GetFacilities()` and simplify some of the changes later in the function.